test/users.test.js

```diff
@@ -8,45 +14,106 @@ const mockedUser = {
 };
 
 describe('users api tests', () => {
-  test('createUser with valid input and the user does not exist creates correctly', () =>
-    expect(createUser(mockedUser)).resolves.toMatch(/manuel.tuero@wolox.com.ar/));
+  test('create user with valid input and the user does not exist creates correctly', async () => {
+    await expect(User.createWithHashedPassword(mockedUser)).resolves.toMatchObject({
```

Comment:
User.createWithHashedPassword(mockedUser) .then(res => {expect()}) is more readable.